app/containers/message/Urls.js

```diff
@@ -81,10 +88,19 @@ const Url = React.memo(({
 	}
 
 	const onPress = () => openLink(url.url, theme);
-
+	const onLongPress = () => {
+		Clipboard.setString(url.url);
+		if (Platform.os === 'ios') {
+			Vibration.vibrate();
+		} else {
+			Vibration.vibrate(200);
+			ToastAndroid.show('Link copied', ToastAndroid.SHORT);
+		}
+	};
```

Comment:
For first, you don't need to use `ToastAndroid`, we have our own [Toast](https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/containers/Toast.js) Component, you can search on our code how to use this.
I don't think you need to add some vibration when a link was copied, only the Toast is a feedback great.

Reply:
Ok, Thanks for the guidance. :+1: